src/QoiSharp/QoiDecoder.cs

```diff
@@ -14,14 +14,14 @@ public static class QoiDecoder
     /// <param name="data">QOI data</param>
     /// <returns>Decoding result.</returns>
     /// <exception cref="QoiDecodingException">Thrown when data is invalid.</exception>
-    public static QoiImage Decode(byte[] data)
+    public static QoiImage Decode(ReadOnlySpan<byte> data)
```

Comment:
Technically this is an unnecessary API breakage as you can keep a `byte[]` and `ReadOnlySpan<byte>` overload. However, if https://github.com/NUlliiON/QoiSharp/pull/2/files#diff-789efe1cf4aa4a90da5c3a8ab571c16736c042985b84858c275d733175a6bc7eR13 is acceptable then the API is broken anyway, so why not clean up an unnecessary API at the same time.

byte[] implicitly converts to ReadOnlySpan<byte> !